zbus/src/connection/mod.rs

```diff
@@ -1863,4 +1882,41 @@ mod p2p_tests {
 
         (conn1, conn2)
     }
+
+    #[test]
+    #[timeout(15000)]
+    fn test_method_timeout() {
+        crate::utils::block_on(test_method_timeout_()).unwrap();
+    }
+
+    #[cfg(unix)]
+    async fn test_method_timeout_() -> Result<()> {
+        #[cfg(not(feature = "tokio"))]
+        use std::os::unix::net::UnixStream;
+        #[cfg(feature = "tokio")]
+        use tokio::net::UnixStream;
+        #[cfg(all(windows, not(feature = "tokio")))]
+        use uds_windows::UnixStream;
+
+        let guid = Guid::generate();
+
+        let (p0, p1) = UnixStream::pair().unwrap();
+
+        #[allow(unused)]
```

Comment:
Please prefix names with `_` instead. In the future the code might change and an unused warning could actually be pointing to an issue and we'll miss it with this around. Hence the preference for `_` prefixes.